src/components/admin/AdminPublishers.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { createRefetchContainer, graphql } from 'react-relay';
+
+import {
+    Grid,
+    Table,
+    Col,
+    Button,
+    Glyphicon,
+    Tooltip,
+    OverlayTrigger,
+    Row
+} from 'react-bootstrap';
+
+
+class AdminPublishers extends React.Component {
+    static propTypes = {
+        viewer: PropTypes.object.isRequired,
+        relay: PropTypes.object.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.adminPublishersRowMapper = this.adminPublishersRowMapper.bind(this);
+        this.refetch = this.refetch.bind(this);
+        this.clearDelete = this.clearDelete.bind(this);
```

Comment:
I would remove these extra methods and the state bootstrapping. That would be needed if we have actions on the publishers but that hasn't been requested yet.